schwifty/bank_registry/manual_pt.json

```diff
@@ -126,5 +126,45 @@
       "name": "BANKINTER, SA - SUCURSAL EM PORTUGAL",
       "primary": true,
       "short_name": "BANKINTER"
+    },
+    {
+      "bank_code": "5180",
+      "bic": "CDCTPTP2",
+      "country_code": "PT",
+      "name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE LEIRIA, CRL",
+      "primary": true,
+      "short_name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE LEIRIA, CRL"
+    },
+    {
+      "bank_code": "5200",
+      "bic": "CDOTPTPP",
+      "country_code": "PT",
+      "name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE MAFRA, CRL",
+      "primary": true,
+      "short_name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE MAFRA, CRL"
+    },
+    {
+      "bank_code": "5340",
+      "bic": "CTIUPTPP",
+      "country_code": "PT",
+      "name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE TORRES VEDRAS, C.R.L",
+      "primary": true,
+      "short_name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE TORRES VEDRAS, C.R.L"
```

Comment:
their site is ccamtv, perhaps we can use this one as short name? still just an idea, ignore it freely

Reply:
I'd keep it the same and avoid any acronyms but if it ends up being the official name, we can rename then! 🤝